app/javascript/react/components/Map/Markers/FixedMarkers.tsx

```diff
@@ -87,6 +84,11 @@ export function FixedMarkers({
 
   const memoizedSessions = useMemo(() => sessions, [sessions]);
 
+  useEffect(() => {
+    console.log("url stream Id", selectedStreamId);
```

Comment:
Do we need this useEffect with console.logs?

Reply:
that's a debugging leftorver, removed it -> thanks for catching that